fawltydeps/traverse_project.py

```diff
@@ -51,13 +77,29 @@ def find_sources(  # pylint: disable=too-many-branches
         else:  # must traverse directory
             traversal.add(path, DepsSource)
 
+    for path in settings.pyenvs if PyEnvSource in source_types else []:
```

Comment:
Nit:
Following rule of 3, handling of source in from the `settings` object could be abstracted out. 
On the other hand we do not anticipate more source types. I will leave it to your discretion :)

Reply:
On yet another hand (;)) it may be easier to test it when it is a separate function.